orchestrator/test_runner/Cargo.toml

```diff
@@ -10,28 +10,25 @@ name = "test-runner"
 path = "src/main.rs"
 
 [dependencies]
+cosmos_gravity = { path = "../cosmos_gravity" }
+ethereum_gravity = { path = "../ethereum_gravity" }
+gravity_proto = { path = "../gravity_proto/" }
+gravity_utils = { path = "../gravity_utils" }
+orchestrator = { path = "../orchestrator/" }
+
 bytes = "1"
-clarity = "0.5"
-cosmos_gravity = {path = "../cosmos_gravity"}
-deep_space = "2.8"
 docopt = "1"
 env_logger = "0.9"
-ethereum_gravity = {path = "../ethereum_gravity"}
 futures = "0.3"
-gravity_proto = {path = "../gravity_proto/"}
-gravity_utils = {path = "../gravity_utils"}
 lazy_static = "1"
 log = "0.4"
 num = "0.4.0"
 num256 = "0.3"
-orchestrator = {path = "../orchestrator/"}
 prost = "0.9"
 prost-types = "0.9"
 rand = "0.8"
-serde = "1.0"
-serde_derive = "1.0"
+serde = { version = "1", features = ["derive"] }
 serde_json = "1.0"
-tokio = {version = "1.4", features = ["macros", "rt-multi-thread"]}
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```

Comment:
why the "tokio" version is "1" instead of "1.4" ?

Reply:
"1" is different from "1.0", "1.0" would get minor version 0 but "1" gets the latest minor version. After this change the Cargo.lock version is `1.17.0`. Tokio `1.4` is really old, see how many fixes there have been: https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md.

Reply:
I have done it